packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -435,7 +435,7 @@ export const GiantTrackTile = ({
       isArtistPick,
       isUnlisted,
       includeEmbed: !(isUnlisted || isStreamGated),
-      includeArtistPick: !isUnlisted,
+      includeArtistPick: true,
```

Comment:
do we not want to exclude unlisted tracks still?

Reply:
chatted with sammie and we want to allow users to set artist pick on a hidden/scheduled track